src/workers/postBannedRep.ts

```diff
@@ -32,14 +32,15 @@ const worker: Worker = {
           }),
         );
 
-        const ownerProps = {
-          targetId: id,
-          targetType: ReputationType.Post,
-          reason: ReputationReason.PostBanned,
-        };
-
         // Skipping penalizing community picks during the beta phase
-        if (data.post.sourceId !== COMMUNITY_PICKS_SOURCE) {
+        // Skipping penalizing shared post to a Squad
+        if (data.post.sourceId !== COMMUNITY_PICKS_SOURCE && !sharedPostId) {
```

Comment:
This is not the right way to check for shared post. Please use the `type` property in the post object

Reply:
I have updated this now and created an enum.